### ISSUE TITLE
 Error Code: 1290. The MySQL server is running with the --secure-file-priv option so it cannot execute this statement

### ISSUE THREAD

**Issue by duttashi** [duttashi/dashboard-database#12]
When trying to export data from mysql using followig code,

`select * from logbook.tbl_grocery into outfile '/grocery_logbook.csv' fields terminated by ',' enclosed by '"' lines terminated by '\n';`

generated the error, ` Error Code: 1290. The MySQL server is running with the --secure-file-priv option so it cannot execute this statement`

**Comment by duttashi** [comment]
I found the solution on this [SO thread](https://stackoverflow.com/questions/31951468/error-code-1290-the-mysql-server-is-running-with-the-secure-file-priv-option)

Solution 1: Use the `secure-file-priv` location. I'm using Windows 10. So I accessed `my.ini` file by launching the `run command window` (press the window key + r key together on the keyboard. In the Run box type `%programdata% and click OK button. The ProgramData window will open up). Here navigate to the location, `\MySQL\MySQL Server 8.0`. And you'll see the `my.ini` file. Open it with Notepad++ and search for `secure-file-priv`. You'll see that it has a value of `secure-file-priv="C:/ProgramData/MySQL/MySQL Server 8.0/Uploads"`.

Now either we can provide the destination path in the select statement like, `select * from logbook.tbl_grocery into outfile 'C:\\ProgramData\\MySQL\\MySQL Server 8.0\\Uploads\\grocery_logbook.csv' fields terminated by ',' enclosed by '"' lines terminated by '\n';`

OR  change the path in my.ini file.

I chose to provide the path in select statement.
